libyul/optimiser/NameDispenser.cpp

```diff
@@ -60,7 +57,7 @@ bool NameDispenser::illegalName(YulString _name)
 {
 	if (_name.empty() || m_usedNames.count(_name) || m_dialect.builtin(_name))
 		return true;
-	if (dynamic_cast<EVMDialect const*>(&m_dialect))
-		return Parser::instructions().count(_name.str());
+	if (auto dialect = dynamic_cast<EVMDialect const*>(&m_dialect))
+		return dialect->builtin(_name) != nullptr;
```

Comment:
That's already part of the case two lines up isn't it?

Reply:
Answered above. I think the discussion is the same for both cases.